Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/basic_block.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/basic_block.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/basic_block.h
@@ -22,7 +22,7 @@
 #include <vector>
 #undef max
 
-#include "third_party/absl/container/btree_map.h"
+#include "absl/container/btree_map.h"
 #include "third_party/zynamics/binexport/instruction.h"
 #include "third_party/zynamics/binexport/util/nested_iterator.h"
 #include "third_party/zynamics/binexport/util/range.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binaryninja/log_sink.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/binaryninja/log_sink.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binaryninja/log_sink.cc
@@ -18,9 +18,9 @@
 #include "binaryninjaapi.h"  // NOLINT
 // clang-format on
 
-#include "third_party/absl/base/log_severity.h"
-#include "third_party/absl/log/log_entry.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/base/log_severity.h"
+#include "absl/log/log_entry.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/logging.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binaryninja/log_sink.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/binaryninja/log_sink.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binaryninja/log_sink.h
@@ -15,8 +15,8 @@
 #ifndef BINARYNINJA_LOG_SINK_H_
 #define BINARYNINJA_LOG_SINK_H_
 
-#include "third_party/absl/log/log_entry.h"
-#include "third_party/absl/log/log_sink.h"
+#include "absl/log/log_entry.h"
+#include "absl/log/log_sink.h"
 #include "third_party/zynamics/binexport/util/logging.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binaryninja/main_plugin.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/binaryninja/main_plugin.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binaryninja/main_plugin.cc
@@ -17,13 +17,13 @@
 #include <cstdint>
 #include <string>
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/escaping.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
+#include "absl/log/log.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/escaping.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
 #include "third_party/zynamics/binexport/binaryninja/log_sink.h"
 #include "third_party/zynamics/binexport/binexport2_writer.h"
 #include "third_party/zynamics/binexport/call_graph.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binexport.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/binexport.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binexport.cc
@@ -16,8 +16,8 @@
 
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
 
 namespace security::binexport {
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binexport2_writer.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/binexport2_writer.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/binexport2_writer.cc
@@ -39,14 +39,14 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/container/flat_hash_map.h"
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/string_view.h"  // IWYU pragma: keep
-#include "third_party/absl/time/clock.h"
-#include "third_party/absl/time/time.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"  // IWYU pragma: keep
+#include "absl/time/clock.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/address_references.h"
 #include "third_party/zynamics/binexport/basic_block.h"
 #include "third_party/zynamics/binexport/binexport2.pb.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/call_graph.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/call_graph.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/call_graph.h
@@ -21,9 +21,9 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/container/btree_set.h"
-#include "third_party/absl/container/node_hash_map.h"
-#include "third_party/absl/container/node_hash_set.h"
+#include "absl/container/btree_set.h"
+#include "absl/container/node_hash_map.h"
+#include "absl/container/node_hash_set.h"
 #include "third_party/zynamics/binexport/comment.h"
 #include "third_party/zynamics/binexport/library_manager.h"
 #include "third_party/zynamics/binexport/util/types.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/comment.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/comment.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/comment.cc
@@ -16,8 +16,8 @@
 
 #include <string>
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
 bool SortComments(const Comment& lhs, const Comment& rhs) {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/edge.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/edge.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/edge.cc
@@ -14,7 +14,7 @@
 
 #include "third_party/zynamics/binexport/edge.h"
 
-#include "third_party/absl/log/log.h"
+#include "absl/log/log.h"
 
 bool operator<(const FlowGraphEdge& one, const FlowGraphEdge& two) {
   if (one.source == two.source) {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/entry_point.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/entry_point.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/entry_point.cc
@@ -17,7 +17,7 @@
 #include <ios>
 #include <string>
 
-#include "third_party/absl/log/log.h"
+#include "absl/log/log.h"
 
 EntryPoint::EntryPoint(Address address, EntryPoint::Source source,
                        const int flags)
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/expression.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/expression.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/expression.cc
@@ -19,7 +19,7 @@
 #include <string>
 #include <utility>
 
-#include "third_party/absl/log/check.h"
+#include "absl/log/check.h"
 #include "third_party/zynamics/binexport/instruction.h"
 #include "third_party/zynamics/binexport/util/hash.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/expression.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/expression.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/expression.h
@@ -20,10 +20,10 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/node_hash_map.h"
-#include "third_party/absl/container/node_hash_set.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/container/node_hash_map.h"
+#include "absl/container/node_hash_set.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 #pragma pack(push, 1)
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/flow_graph.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/flow_graph.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/flow_graph.cc
@@ -25,10 +25,10 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/container/flat_hash_set.h"
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/container/flat_hash_set.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/function.h"
 #include "third_party/zynamics/binexport/util/format.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/flow_graph.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/flow_graph.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/flow_graph.h
@@ -21,8 +21,8 @@
 #include <tuple>
 #include <vector>
 
-#include "third_party/absl/container/btree_map.h"
-#include "third_party/absl/container/node_hash_set.h"
+#include "absl/container/btree_map.h"
+#include "absl/container/node_hash_set.h"
 #include "third_party/zynamics/binexport/edge.h"
 #include "third_party/zynamics/binexport/function.h"
 #include "third_party/zynamics/binexport/instruction.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/function.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/function.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/function.cc
@@ -24,9 +24,9 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/function.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/function.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/function.h
@@ -20,8 +20,8 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/btree_map.h"
-#include "third_party/absl/container/node_hash_set.h"
+#include "absl/container/btree_map.h"
+#include "absl/container/node_hash_set.h"
 #include "third_party/zynamics/binexport/basic_block.h"
 #include "third_party/zynamics/binexport/edge.h"
 #include "third_party/zynamics/binexport/instruction.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/arm.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/arm.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/arm.cc
@@ -26,8 +26,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/dalvik.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/dalvik.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/dalvik.cc
@@ -31,8 +31,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/digest.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/digest.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/digest.cc
@@ -22,9 +22,9 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/escaping.h"
+#include "absl/status/status.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/escaping.h"
 
 namespace security::binexport {
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/digest.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/digest.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/digest.h
@@ -17,7 +17,7 @@
 
 #include <string>
 
-#include "third_party/absl/status/statusor.h"
+#include "absl/status/statusor.h"
 
 namespace security::binexport {
 //
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/flow_analysis.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/flow_analysis.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/flow_analysis.cc
@@ -41,10 +41,10 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/time/time.h"
+#include "absl/log/log.h"
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/address_references.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/edge.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/flow_analysis.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/flow_analysis.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/flow_analysis.h
@@ -15,7 +15,7 @@
 #ifndef IDA_FLOW_ANALYSIS_H_
 #define IDA_FLOW_ANALYSIS_H_
 
-#include "third_party/absl/container/btree_map.h"
+#include "absl/container/btree_map.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/entry_point.h"
 #include "third_party/zynamics/binexport/expression.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/generic.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/generic.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/generic.cc
@@ -24,8 +24,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/log_sink.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/log_sink.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/log_sink.cc
@@ -22,8 +22,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log_entry.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/log/log_entry.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/logging.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/log_sink.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/log_sink.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/log_sink.h
@@ -15,8 +15,8 @@
 #ifndef IDA_LOG_SINK_H_
 #define IDA_LOG_SINK_H_
 
-#include "third_party/absl/log/log_entry.h"
-#include "third_party/absl/log/log_sink.h"
+#include "absl/log/log_entry.h"
+#include "absl/log/log_sink.h"
 #include "third_party/zynamics/binexport/util/logging.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/main_plugin.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/main_plugin.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/main_plugin.cc
@@ -32,15 +32,15 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/base/attributes.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/escaping.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/numbers.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/time/time.h"
+#include "absl/base/attributes.h"
+#include "absl/log/log.h"
+#include "absl/memory/memory.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/escaping.h"
+#include "absl/strings/match.h"
+#include "absl/strings/numbers.h"
+#include "absl/strings/str_cat.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/binexport2_writer.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/dump_writer.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/metapc.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/metapc.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/metapc.cc
@@ -27,8 +27,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/base_types.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/flow_graph.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/mips.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/mips.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/mips.cc
@@ -27,8 +27,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 #include "third_party/zynamics/binexport/util/format.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/names.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/names.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/names.cc
@@ -45,11 +45,11 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/container/btree_map.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/time/time.h"
+#include "absl/container/btree_map.h"
+#include "absl/log/log.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/str_cat.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/address_references.h"
 #include "third_party/zynamics/binexport/base_types.h"
 #include "third_party/zynamics/binexport/call_graph.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/names.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/names.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/names.h
@@ -25,7 +25,7 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/types/optional.h"
+#include "absl/types/optional.h"
 #include "third_party/zynamics/binexport/comment.h"
 #include "third_party/zynamics/binexport/expression.h"
 #include "third_party/zynamics/binexport/util/types.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/ppc.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/ppc.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/ppc.cc
@@ -26,8 +26,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 #include "third_party/zynamics/binexport/instruction.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/ui.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/ui.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/ui.cc
@@ -31,11 +31,11 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/types/span.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"
+#include "absl/types/span.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/util/process.h"
 #include "third_party/zynamics/binexport/util/types.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/ui.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/ui.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/ui.h
@@ -24,10 +24,10 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/types/span.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+#include "absl/types/span.h"
 
 // Small RAII class that displays a wait message for long-running actions.
 class WaitBox {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/util.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/util.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/util.cc
@@ -14,7 +14,7 @@
 
 #include "third_party/zynamics/binexport/ida/util.h"
 
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/string_view.h"
 
 namespace security::binexport {
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/util.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/ida/util.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/ida/util.h
@@ -22,7 +22,7 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/string_view.h"
 
 namespace security::binexport {
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/instruction.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/instruction.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/instruction.cc
@@ -26,8 +26,8 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
 #include "third_party/zynamics/binexport/flow_graph.h"
 #include "third_party/zynamics/binexport/operand.h"
 #include "third_party/zynamics/binexport/virtual_memory.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/instruction.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/instruction.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/instruction.h
@@ -21,7 +21,7 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/node_hash_set.h"
+#include "absl/container/node_hash_set.h"
 #include "third_party/zynamics/binexport/operand.h"
 #include "third_party/zynamics/binexport/util/range.h"
 #include "third_party/zynamics/binexport/util/types.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/library_manager.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/library_manager.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/library_manager.cc
@@ -17,7 +17,7 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
+#include "absl/log/check.h"
 
 int LibraryManager::AddKnownLibrary(const std::string& library_name,
                                     Linkage linkage) {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/library_manager.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/library_manager.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/library_manager.h
@@ -19,7 +19,7 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_map.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 // Theory of operations: this class keeps a list of currently known libraries,
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/operand.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/operand.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/operand.cc
@@ -17,7 +17,7 @@
 #include <cstdint>
 #include <string>
 
-#include "third_party/absl/log/check.h"
+#include "absl/log/check.h"
 
 thread_local Expressions Operand::expressions_;
 thread_local Operand::OperandCache Operand::operand_cache_;
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/operand.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/operand.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/operand.h
@@ -18,8 +18,8 @@
 #include <cstdint>
 #include <string>
 
-#include "third_party/absl/container/flat_hash_set.h"
-#include "third_party/absl/container/node_hash_map.h"
+#include "absl/container/flat_hash_set.h"
+#include "absl/container/node_hash_map.h"
 #include "third_party/zynamics/binexport/expression.h"
 
 #pragma pack(push, 1)
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/call_graph.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/reader/call_graph.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/call_graph.cc
@@ -18,9 +18,9 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/check.h"
+#include "absl/memory/memory.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/reader/graph_utility.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/call_graph_test.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/reader/call_graph_test.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/call_graph_test.cc
@@ -18,7 +18,7 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/reader/graph_utility.h"
 #include "third_party/zynamics/binexport/testing.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/flow_graph.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/reader/flow_graph.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/flow_graph.cc
@@ -23,14 +23,14 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/types/optional.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
+#include "absl/memory/memory.h"
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/types/optional.h"
 #include "third_party/zynamics/binexport/architectures.h"
-#include "third_party/absl/strings/ascii.h"
+#include "absl/strings/ascii.h"
 #include "third_party/zynamics/binexport/reader/graph_utility.h"
 #include "third_party/zynamics/binexport/reader/instruction.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/flow_graph.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/reader/flow_graph.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/flow_graph.h
@@ -28,8 +28,8 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/base/attributes.h"
-#include "third_party/absl/types/optional.h"
+#include "absl/base/attributes.h"
+#include "absl/types/optional.h"
 #include "third_party/zynamics/binexport/architectures.h"
 #include "third_party/zynamics/binexport/binexport2.pb.h"
 #include "third_party/zynamics/binexport/reader/instruction.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/instruction.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/reader/instruction.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/reader/instruction.h
@@ -22,8 +22,8 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/inlined_vector.h"
-#include "third_party/absl/types/optional.h"
+#include "absl/container/inlined_vector.h"
+#include "absl/types/optional.h"
 #include "third_party/zynamics/binexport/architectures.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/statistics_writer.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/statistics_writer.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/statistics_writer.cc
@@ -19,7 +19,7 @@
 #include <ostream>
 #include <string>
 
-#include "third_party/absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_map.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/flow_graph.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/statistics_writer.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/statistics_writer.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/statistics_writer.h
@@ -17,7 +17,7 @@
 
 #include <fstream>
 
-#include "third_party/absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_map.h"
 #include "third_party/zynamics/binexport/writer.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/testing.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/testing.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/testing.cc
@@ -4,8 +4,8 @@
 #include <fstream>
 
 #include "gtest/gtest.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/testing.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/testing.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/testing.h
@@ -17,8 +17,8 @@
 
 #include <string>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/binexport2.pb.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/binexport2dump.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/tools/binexport2dump.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/binexport2dump.cc
@@ -19,17 +19,17 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/cleanup/cleanup.h"
-#include "third_party/absl/container/flat_hash_map.h"
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/initialize.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/time/time.h"
+#include "absl/cleanup/cleanup.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/log/check.h"
+#include "absl/log/initialize.h"
+#include "absl/log/log.h"
+#include "absl/status/status.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/binexport.h"
 #include "third_party/zynamics/binexport/binexport2.pb.h"
 #include "third_party/zynamics/binexport/tools/command_util.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/bxp_dummy.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/tools/bxp_dummy.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/bxp_dummy.cc
@@ -17,10 +17,10 @@
 
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/tools/command_util.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/bxp_main.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/tools/bxp_main.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/bxp_main.cc
@@ -34,12 +34,12 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/flags/parse.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/flags/parse.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/tools/command_util.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/util/process.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/command_util.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/tools/command_util.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/command_util.cc
@@ -19,14 +19,14 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/flags/parse.h"
-#include "third_party/absl/flags/usage.h"
-#include "third_party/absl/flags/usage_config.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/flags/parse.h"
+#include "absl/flags/usage.h"
+#include "absl/flags/usage_config.h"
+#include "absl/status/status.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/version.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/command_util.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/tools/command_util.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/tools/command_util.h
@@ -18,8 +18,8 @@
 #include <functional>
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/strings/string_view.h"
 
 namespace security::binexport {
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/filesystem.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/filesystem.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/filesystem.cc
@@ -51,13 +51,13 @@
 #include <system_error>  // NOLINT(build/c++11)
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_replace.h"  // IWYU pragma: keep
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/strings/strip.h"  // IWYU pragma: keep
+#include "absl/status/status.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_replace.h"  // IWYU pragma: keep
+#include "absl/strings/string_view.h"
+#include "absl/strings/strip.h"  // IWYU pragma: keep
 #include "third_party/zynamics/binexport/util/process.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/filesystem.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/filesystem.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/filesystem.h
@@ -20,9 +20,9 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 #ifdef _WIN32
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/filesystem_test.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/filesystem_test.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/filesystem_test.cc
@@ -19,8 +19,8 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/status/status_matchers.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/status/status_matchers.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/format.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/format.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/format.cc
@@ -4,9 +4,9 @@
 #include <cinttypes>
 #include <cstdio>
 
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/time/time.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/time/time.h"
 
 namespace security::binexport {
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/format.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/format.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/format.h
@@ -17,7 +17,7 @@
 
 #include <string>
 
-#include "third_party/absl/time/time.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/idb_export.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/idb_export.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/idb_export.cc
@@ -18,20 +18,20 @@
 #include <memory>
 #include <thread>  // NOLINT
 
-#include "third_party/absl/base/attributes.h"
-#include "third_party/absl/container/flat_hash_map.h"
-#include "third_party/absl/container/flat_hash_set.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/str_join.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/synchronization/mutex.h"
-#include "third_party/absl/time/time.h"
+#include "absl/base/attributes.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_set.h"
+#include "absl/memory/memory.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/str_join.h"
+#include "absl/strings/string_view.h"
+#include "absl/synchronization/mutex.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/util/process.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/idb_export.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/idb_export.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/idb_export.h
@@ -18,10 +18,10 @@
 #include <memory>
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/synchronization/mutex.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+#include "absl/synchronization/mutex.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/logging.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/logging.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/logging.h
@@ -20,9 +20,9 @@
 #include <string>
 #include <thread>  // NOLINT
 
-#include "third_party/absl/base/log_severity.h"
-#include "third_party/absl/log/log_sink.h"
-#include "third_party/absl/status/status.h"
+#include "absl/base/log_severity.h"
+#include "absl/log/log_sink.h"
+#include "absl/status/status.h"
 
 namespace security::binexport {
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/process.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/process.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/process.cc
@@ -43,11 +43,11 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/base/attributes.h"  // IWYU pragma: keep
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_replace.h"  // IWYU pragma: keep
+#include "absl/base/attributes.h"  // IWYU pragma: keep
+#include "absl/status/status.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_replace.h"  // IWYU pragma: keep
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
 
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/process.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/process.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/process.h
@@ -18,9 +18,9 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/process_test.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/process_test.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/process_test.cc
@@ -19,8 +19,8 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/status/status_matchers.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/status/status_matchers.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
 
 namespace security::binexport {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/status_macros.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/status_macros.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/status_macros.h
@@ -18,9 +18,9 @@
 #ifndef UTIL_STATUS_MACROS_H_
 #define UTIL_STATUS_MACROS_H_
 
-#include "third_party/absl/base/optimization.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
+#include "absl/base/optimization.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
 
 // Internal helper for concatenating macro values.
 #define NA_MACROS_IMPL_CONCAT_INNER_(x, y) x##y
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/status_macros_test.cc
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/util/status_macros_test.cc
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/util/status_macros_test.cc
@@ -18,11 +18,11 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/status_matchers.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/memory/memory.h"
+#include "absl/status/status.h"
+#include "absl/status/status_matchers.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/str_cat.h"
 
 namespace not_absl {
 namespace {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/virtual_memory.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/virtual_memory.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/virtual_memory.h
@@ -19,8 +19,8 @@
 #include <map>
 #include <vector>
 
-#include "third_party/absl/container/btree_map.h"
-#include "third_party/absl/container/flat_hash_map.h"
+#include "absl/container/btree_map.h"
+#include "absl/container/flat_hash_map.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 class AddressSpace {
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/writer.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/writer.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/writer.h
@@ -15,7 +15,7 @@
 #ifndef WRITER_H_
 #define WRITER_H_
 
-#include "third_party/absl/status/status.h"
+#include "absl/status/status.h"
 #include "third_party/zynamics/binexport/address_references.h"
 #include "third_party/zynamics/binexport/comment.h"
 #include "third_party/zynamics/binexport/instruction.h"
Index: binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/x86_nop.h
===================================================================
--- binexport-644786690fcb6bcf6d42a76e7d8b65056176054d.orig/x86_nop.h
+++ binexport-644786690fcb6bcf6d42a76e7d8b65056176054d/x86_nop.h
@@ -17,7 +17,7 @@
 
 #include <cstddef>
 
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/string_view.h"
 
 // Returns true if the instruction starting at the first byte in "bytes" is a
 // NOP instruction. NOPs have been taken from the GNU assembler
