diff --git a/basic_block.h b/basic_block.h
index 7b300e4..561d9a0 100644
--- a/basic_block.h
+++ b/basic_block.h
@@ -22,7 +22,7 @@
 #include <vector>
 #undef max
 
-#include "third_party/absl/container/btree_map.h"
+#include "absl/container/btree_map.h"
 #include "third_party/zynamics/binexport/instruction.h"
 #include "third_party/zynamics/binexport/util/nested_iterator.h"
 #include "third_party/zynamics/binexport/util/range.h"
diff --git a/binaryninja/log_sink.cc b/binaryninja/log_sink.cc
index bd2225f..439af69 100644
--- a/binaryninja/log_sink.cc
+++ b/binaryninja/log_sink.cc
@@ -18,8 +18,8 @@
 #include "binaryninjaapi.h"  // NOLINT
 // clang-format on
 
-#include "third_party/absl/base/log_severity.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/base/log_severity.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/logging.h"
 
 namespace security::binexport {
diff --git a/binaryninja/log_sink.h b/binaryninja/log_sink.h
index 97adb2c..265c247 100644
--- a/binaryninja/log_sink.h
+++ b/binaryninja/log_sink.h
@@ -15,7 +15,7 @@
 #ifndef BINARYNINJA_LOG_SINK_H_
 #define BINARYNINJA_LOG_SINK_H_
 
-#include "third_party/absl/log/log_sink.h"
+#include "absl/log/log_sink.h"
 #include "third_party/zynamics/binexport/util/logging.h"
 
 namespace security::binexport {
diff --git a/binaryninja/main_plugin.cc b/binaryninja/main_plugin.cc
index 78d5cea..d61064f 100644
--- a/binaryninja/main_plugin.cc
+++ b/binaryninja/main_plugin.cc
@@ -17,13 +17,13 @@
 #include <cstdint>
 #include <string>
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/escaping.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
+#include "absl/log/log.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/escaping.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
 #include "third_party/zynamics/binexport/binaryninja/log_sink.h"
 #include "third_party/zynamics/binexport/binexport2_writer.h"
 #include "third_party/zynamics/binexport/call_graph.h"
diff --git a/binexport.cc b/binexport.cc
index c42d041..fb26447 100644
--- a/binexport.cc
+++ b/binexport.cc
@@ -16,8 +16,8 @@
 
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
 
 namespace security::binexport {
 
diff --git a/binexport2_writer.cc b/binexport2_writer.cc
index 48d00a6..c66a71a 100644
--- a/binexport2_writer.cc
+++ b/binexport2_writer.cc
@@ -39,12 +39,12 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/container/flat_hash_map.h"
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/time/clock.h"
-#include "third_party/absl/time/time.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
+#include "absl/time/clock.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/binexport2.pb.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/flow_graph.h"
diff --git a/call_graph.h b/call_graph.h
index daa5ac5..ee05f05 100644
--- a/call_graph.h
+++ b/call_graph.h
@@ -17,9 +17,9 @@
 
 #include <iosfwd>
 
-#include "third_party/absl/container/btree_set.h"
-#include "third_party/absl/container/node_hash_map.h"
-#include "third_party/absl/container/node_hash_set.h"
+#include "absl/container/btree_set.h"
+#include "absl/container/node_hash_map.h"
+#include "absl/container/node_hash_set.h"
 #include "third_party/zynamics/binexport/comment.h"
 #include "third_party/zynamics/binexport/library_manager.h"
 #include "third_party/zynamics/binexport/util/types.h"
diff --git a/comment.cc b/comment.cc
index 5f48028..ba72141 100644
--- a/comment.cc
+++ b/comment.cc
@@ -16,8 +16,8 @@
 
 #include <string>
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
 bool SortComments(const Comment& lhs, const Comment& rhs) {
diff --git a/edge.cc b/edge.cc
index 6def1de..ef669c2 100644
--- a/edge.cc
+++ b/edge.cc
@@ -16,7 +16,7 @@
 
 #include <functional>
 
-#include "third_party/absl/log/log.h"
+#include "absl/log/log.h"
 
 bool operator<(const FlowGraphEdge& one, const FlowGraphEdge& two) {
   if (one.source == two.source) {
diff --git a/entry_point.cc b/entry_point.cc
index aaf1512..cfafc5a 100644
--- a/entry_point.cc
+++ b/entry_point.cc
@@ -17,7 +17,7 @@
 #include <ios>
 #include <string>
 
-#include "third_party/absl/log/log.h"
+#include "absl/log/log.h"
 
 EntryPoint::EntryPoint(Address address, EntryPoint::Source source,
                        const int flags)
diff --git a/expression.cc b/expression.cc
index b237799..0e332f1 100644
--- a/expression.cc
+++ b/expression.cc
@@ -19,7 +19,7 @@
 #include <string>
 #include <utility>
 
-#include "third_party/absl/log/check.h"
+#include "absl/log/check.h"
 #include "third_party/zynamics/binexport/instruction.h"
 #include "third_party/zynamics/binexport/util/hash.h"
 
diff --git a/expression.h b/expression.h
index 7af7ee2..e15ee39 100644
--- a/expression.h
+++ b/expression.h
@@ -20,10 +20,10 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/node_hash_map.h"
-#include "third_party/absl/container/node_hash_set.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/container/node_hash_map.h"
+#include "absl/container/node_hash_set.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 #pragma pack(push, 1)
diff --git a/flow_graph.cc b/flow_graph.cc
index 8d25cd3..b130548 100644
--- a/flow_graph.cc
+++ b/flow_graph.cc
@@ -25,10 +25,10 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/container/flat_hash_set.h"
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/container/flat_hash_set.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/function.h"
 #include "third_party/zynamics/binexport/util/format.h"
diff --git a/flow_graph.h b/flow_graph.h
index a59effa..74a1078 100644
--- a/flow_graph.h
+++ b/flow_graph.h
@@ -21,8 +21,8 @@
 #include <tuple>
 #include <vector>
 
-#include "third_party/absl/container/btree_map.h"
-#include "third_party/absl/container/node_hash_set.h"
+#include "absl/container/btree_map.h"
+#include "absl/container/node_hash_set.h"
 #include "third_party/zynamics/binexport/edge.h"
 #include "third_party/zynamics/binexport/function.h"
 #include "third_party/zynamics/binexport/instruction.h"
diff --git a/function.cc b/function.cc
index a64c861..8d2bdae 100644
--- a/function.cc
+++ b/function.cc
@@ -24,9 +24,9 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
diff --git a/function.h b/function.h
index 52f7b2f..ec74612 100644
--- a/function.h
+++ b/function.h
@@ -19,8 +19,8 @@
 #include <ostream>
 #include <vector>
 
-#include "third_party/absl/container/btree_map.h"
-#include "third_party/absl/container/node_hash_set.h"
+#include "absl/container/btree_map.h"
+#include "absl/container/node_hash_set.h"
 #include "third_party/zynamics/binexport/basic_block.h"
 #include "third_party/zynamics/binexport/edge.h"
 #include "third_party/zynamics/binexport/util/types.h"
diff --git a/ida/arm.cc b/ida/arm.cc
index 31a2b12..84eb642 100644
--- a/ida/arm.cc
+++ b/ida/arm.cc
@@ -26,8 +26,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
diff --git a/ida/dalvik.cc b/ida/dalvik.cc
index d629c42..6e92135 100644
--- a/ida/dalvik.cc
+++ b/ida/dalvik.cc
@@ -30,8 +30,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
diff --git a/ida/digest.cc b/ida/digest.cc
index fa16dfc..155199c 100644
--- a/ida/digest.cc
+++ b/ida/digest.cc
@@ -21,9 +21,9 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/escaping.h"
+#include "absl/status/status.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/escaping.h"
 
 namespace security::binexport {
 
diff --git a/ida/digest.h b/ida/digest.h
index 1b3d2b2..835515e 100644
--- a/ida/digest.h
+++ b/ida/digest.h
@@ -17,7 +17,7 @@
 
 #include <string>
 
-#include "third_party/absl/status/statusor.h"
+#include "absl/status/statusor.h"
 
 namespace security::binexport {
 //
diff --git a/ida/flow_analysis.cc b/ida/flow_analysis.cc
index c96febe..e68c119 100644
--- a/ida/flow_analysis.cc
+++ b/ida/flow_analysis.cc
@@ -30,8 +30,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/flow_analysis.h"
 #include "third_party/zynamics/binexport/ida/arm.h"
 #include "third_party/zynamics/binexport/ida/dalvik.h"
diff --git a/ida/flow_analysis.h b/ida/flow_analysis.h
index e237bb3..899ccf4 100644
--- a/ida/flow_analysis.h
+++ b/ida/flow_analysis.h
@@ -15,7 +15,7 @@
 #ifndef IDA_FLOW_ANALYSIS_H_
 #define IDA_FLOW_ANALYSIS_H_
 
-#include "third_party/absl/container/btree_map.h"
+#include "absl/container/btree_map.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/entry_point.h"
 #include "third_party/zynamics/binexport/expression.h"
diff --git a/ida/generic.cc b/ida/generic.cc
index 9bf2a84..e51fe68 100644
--- a/ida/generic.cc
+++ b/ida/generic.cc
@@ -24,8 +24,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 
 namespace security::binexport {
diff --git a/ida/log_sink.cc b/ida/log_sink.cc
index 9ce6eaa..ce3bffd 100644
--- a/ida/log_sink.cc
+++ b/ida/log_sink.cc
@@ -22,7 +22,7 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/logging.h"
 
 namespace security::binexport {
diff --git a/ida/log_sink.h b/ida/log_sink.h
index 5488dea..8064340 100644
--- a/ida/log_sink.h
+++ b/ida/log_sink.h
@@ -15,7 +15,7 @@
 #ifndef IDA_LOG_SINK_H_
 #define IDA_LOG_SINK_H_
 
-#include "third_party/absl/log/log_sink.h"
+#include "absl/log/log_sink.h"
 #include "third_party/zynamics/binexport/util/logging.h"
 
 namespace security::binexport {
diff --git a/ida/main_plugin.cc b/ida/main_plugin.cc
index 131958f..3e96b3c 100644
--- a/ida/main_plugin.cc
+++ b/ida/main_plugin.cc
@@ -25,15 +25,15 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/base/attributes.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/escaping.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/numbers.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/time/time.h"
+#include "absl/base/attributes.h"
+#include "absl/log/log.h"
+#include "absl/memory/memory.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/escaping.h"
+#include "absl/strings/match.h"
+#include "absl/strings/numbers.h"
+#include "absl/strings/str_cat.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/binexport2_writer.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/dump_writer.h"
diff --git a/ida/metapc.cc b/ida/metapc.cc
index 28d0b3a..643e9e7 100644
--- a/ida/metapc.cc
+++ b/ida/metapc.cc
@@ -27,8 +27,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/base_types.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/flow_graph.h"
diff --git a/ida/mips.cc b/ida/mips.cc
index d6f764a..d7471ab 100644
--- a/ida/mips.cc
+++ b/ida/mips.cc
@@ -27,8 +27,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 #include "third_party/zynamics/binexport/util/format.h"
diff --git a/ida/names.cc b/ida/names.cc
index a62f6cd..3f3d367 100644
--- a/ida/names.cc
+++ b/ida/names.cc
@@ -38,11 +38,11 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/container/btree_map.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/time/time.h"
+#include "absl/container/btree_map.h"
+#include "absl/log/log.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/str_cat.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/address_references.h"
 #include "third_party/zynamics/binexport/base_types.h"
 #include "third_party/zynamics/binexport/call_graph.h"
diff --git a/ida/names.h b/ida/names.h
index 42dc32a..97ff57d 100644
--- a/ida/names.h
+++ b/ida/names.h
@@ -25,7 +25,7 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/types/optional.h"
+#include "absl/types/optional.h"
 #include "third_party/zynamics/binexport/comment.h"
 #include "third_party/zynamics/binexport/expression.h"
 #include "third_party/zynamics/binexport/util/types.h"
diff --git a/ida/ppc.cc b/ida/ppc.cc
index b2a6fc4..ce01d68 100644
--- a/ida/ppc.cc
+++ b/ida/ppc.cc
@@ -26,8 +26,8 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/ida/names.h"
 #include "third_party/zynamics/binexport/instruction.h"
diff --git a/ida/ui.cc b/ida/ui.cc
index aad61b0..2e0fb9d 100644
--- a/ida/ui.cc
+++ b/ida/ui.cc
@@ -30,11 +30,11 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/types/span.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"
+#include "absl/types/span.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/util/process.h"
 #include "third_party/zynamics/binexport/util/types.h"
diff --git a/ida/ui.h b/ida/ui.h
index f0b9560..1bc6f31 100644
--- a/ida/ui.h
+++ b/ida/ui.h
@@ -24,10 +24,10 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/types/span.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+#include "absl/types/span.h"
 
 // Small RAII class that displays a wait message for long-running actions.
 class WaitBox {
diff --git a/ida/util.cc b/ida/util.cc
index 9d95545..a7de7d3 100644
--- a/ida/util.cc
+++ b/ida/util.cc
@@ -14,7 +14,7 @@
 
 #include "third_party/zynamics/binexport/ida/util.h"
 
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/string_view.h"
 
 namespace security::binexport {
 
diff --git a/ida/util.h b/ida/util.h
index 62fe75a..cc5a746 100644
--- a/ida/util.h
+++ b/ida/util.h
@@ -22,7 +22,7 @@
 #include "third_party/zynamics/binexport/ida/end_idasdk.inc"    // NOLINT
 // clang-format on
 
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/string_view.h"
 
 namespace security::binexport {
 
diff --git a/instruction.cc b/instruction.cc
index 0ac1baa..24dea4c 100644
--- a/instruction.cc
+++ b/instruction.cc
@@ -26,8 +26,8 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
 #include "third_party/zynamics/binexport/flow_graph.h"
 #include "third_party/zynamics/binexport/operand.h"
 #include "third_party/zynamics/binexport/virtual_memory.h"
diff --git a/instruction.h b/instruction.h
index 03bddd1..5480ec3 100644
--- a/instruction.h
+++ b/instruction.h
@@ -19,7 +19,7 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/node_hash_set.h"
+#include "absl/container/node_hash_set.h"
 #include "third_party/zynamics/binexport/operand.h"
 #include "third_party/zynamics/binexport/util/range.h"
 
diff --git a/library_manager.cc b/library_manager.cc
index 58284b8..3091e53 100644
--- a/library_manager.cc
+++ b/library_manager.cc
@@ -17,7 +17,7 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
+#include "absl/log/check.h"
 
 int LibraryManager::AddKnownLibrary(const std::string& library_name,
                                     Linkage linkage) {
diff --git a/library_manager.h b/library_manager.h
index 3664a3a..4f602c1 100644
--- a/library_manager.h
+++ b/library_manager.h
@@ -19,7 +19,7 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_map.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 // Theory of operations: this class keeps a list of currently known libraries,
diff --git a/operand.cc b/operand.cc
index 600e50e..62e8add 100644
--- a/operand.cc
+++ b/operand.cc
@@ -17,7 +17,7 @@
 #include <cstdint>
 #include <string>
 
-#include "third_party/absl/log/check.h"
+#include "absl/log/check.h"
 
 thread_local Expressions Operand::expressions_;
 thread_local Operand::OperandCache Operand::operand_cache_;
diff --git a/operand.h b/operand.h
index 9d7a37a..f62290c 100644
--- a/operand.h
+++ b/operand.h
@@ -18,8 +18,8 @@
 #include <cstdint>
 #include <string>
 
-#include "third_party/absl/container/flat_hash_set.h"
-#include "third_party/absl/container/node_hash_map.h"
+#include "absl/container/flat_hash_set.h"
+#include "absl/container/node_hash_map.h"
 #include "third_party/zynamics/binexport/expression.h"
 
 #pragma pack(push, 1)
diff --git a/reader/call_graph.cc b/reader/call_graph.cc
index 75b650c..743751a 100644
--- a/reader/call_graph.cc
+++ b/reader/call_graph.cc
@@ -18,9 +18,9 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/check.h"
+#include "absl/memory/memory.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/reader/graph_utility.h"
 #include "third_party/zynamics/binexport/util/format.h"
 
diff --git a/reader/call_graph_test.cc b/reader/call_graph_test.cc
index 6e1f477..d9e84e2 100644
--- a/reader/call_graph_test.cc
+++ b/reader/call_graph_test.cc
@@ -18,7 +18,7 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/reader/graph_utility.h"
 #include "third_party/zynamics/binexport/testing.h"
 
diff --git a/reader/flow_graph.cc b/reader/flow_graph.cc
index 95991a0..44420c5 100644
--- a/reader/flow_graph.cc
+++ b/reader/flow_graph.cc
@@ -20,9 +20,9 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/memory/memory.h"
+#include "absl/log/check.h"
+#include "absl/log/log.h"
+#include "absl/memory/memory.h"
 #include "third_party/zynamics/binexport/reader/graph_utility.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
diff --git a/reader/flow_graph.h b/reader/flow_graph.h
index 75fed1d..e78e64c 100644
--- a/reader/flow_graph.h
+++ b/reader/flow_graph.h
@@ -25,8 +25,8 @@
 #include <utility>
 #include <vector>
 
-#include "third_party/absl/base/macros.h"
-#include "third_party/absl/types/optional.h"
+#include "absl/base/macros.h"
+#include "absl/types/optional.h"
 #include "third_party/zynamics/binexport/architectures.h"
 #include "third_party/zynamics/binexport/binexport2.pb.h"
 #include "third_party/zynamics/binexport/reader/instruction.h"
diff --git a/reader/instruction.h b/reader/instruction.h
index 4848afe..4116934 100644
--- a/reader/instruction.h
+++ b/reader/instruction.h
@@ -22,8 +22,8 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/container/inlined_vector.h"
-#include "third_party/absl/types/optional.h"
+#include "absl/container/inlined_vector.h"
+#include "absl/types/optional.h"
 #include "third_party/zynamics/binexport/architectures.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
diff --git a/statistics_writer.cc b/statistics_writer.cc
index 16fe5d2..7c09d2a 100644
--- a/statistics_writer.cc
+++ b/statistics_writer.cc
@@ -19,7 +19,7 @@
 #include <ostream>
 #include <string>
 
-#include "third_party/absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_map.h"
 #include "third_party/zynamics/binexport/call_graph.h"
 #include "third_party/zynamics/binexport/flow_graph.h"
 
diff --git a/statistics_writer.h b/statistics_writer.h
index 64e1a8b..6ab4221 100644
--- a/statistics_writer.h
+++ b/statistics_writer.h
@@ -17,7 +17,7 @@
 
 #include <fstream>
 
-#include "third_party/absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_map.h"
 #include "third_party/zynamics/binexport/writer.h"
 
 namespace security::binexport {
diff --git a/testing.cc b/testing.cc
index 7a0310a..80945f8 100644
--- a/testing.cc
+++ b/testing.cc
@@ -4,8 +4,8 @@
 #include <fstream>
 
 #include "gtest/gtest.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/log/log.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 
 namespace security::binexport {
diff --git a/testing.h b/testing.h
index e2a35ed..5ec6685 100644
--- a/testing.h
+++ b/testing.h
@@ -17,8 +17,8 @@
 
 #include <string>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/binexport2.pb.h"
 
 namespace security::binexport {
diff --git a/tools/binexport2dump.cc b/tools/binexport2dump.cc
index 96717f4..3ab3e0b 100644
--- a/tools/binexport2dump.cc
+++ b/tools/binexport2dump.cc
@@ -19,17 +19,17 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/cleanup/cleanup.h"
-#include "third_party/absl/container/flat_hash_map.h"
-#include "third_party/absl/log/check.h"
-#include "third_party/absl/log/initialize.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/time/time.h"
+#include "absl/cleanup/cleanup.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/log/check.h"
+#include "absl/log/initialize.h"
+#include "absl/log/log.h"
+#include "absl/status/status.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/binexport.h"
 #include "third_party/zynamics/binexport/binexport2.pb.h"
 #include "third_party/zynamics/binexport/tools/command_util.h"
diff --git a/tools/bxp_dummy.cc b/tools/bxp_dummy.cc
index 20d0ef0..076b926 100644
--- a/tools/bxp_dummy.cc
+++ b/tools/bxp_dummy.cc
@@ -17,10 +17,10 @@
 
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/tools/command_util.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 
diff --git a/tools/bxp_main.cc b/tools/bxp_main.cc
index c5aad3e..472d779 100644
--- a/tools/bxp_main.cc
+++ b/tools/bxp_main.cc
@@ -34,12 +34,12 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/flags/parse.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/flags/parse.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/tools/command_util.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/util/process.h"
diff --git a/tools/command_util.cc b/tools/command_util.cc
index fa62859..bcf0cc7 100644
--- a/tools/command_util.cc
+++ b/tools/command_util.cc
@@ -19,14 +19,14 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/flags/parse.h"
-#include "third_party/absl/flags/usage.h"
-#include "third_party/absl/flags/usage_config.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/flags/parse.h"
+#include "absl/flags/usage.h"
+#include "absl/flags/usage_config.h"
+#include "absl/status/status.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/version.h"
 
diff --git a/tools/command_util.h b/tools/command_util.h
index b853345..0ebd5aa 100644
--- a/tools/command_util.h
+++ b/tools/command_util.h
@@ -18,8 +18,8 @@
 #include <functional>
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/strings/string_view.h"
 
 namespace security::binexport {
 
diff --git a/util/filesystem.cc b/util/filesystem.cc
index 43974d0..0d85abd 100644
--- a/util/filesystem.cc
+++ b/util/filesystem.cc
@@ -51,13 +51,13 @@
 #include <system_error>  // NOLINT(build/c++11)
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_replace.h"  // IWYU pragma: keep
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/strings/strip.h"  // IWYU pragma: keep
+#include "absl/status/status.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_replace.h"  // IWYU pragma: keep
+#include "absl/strings/string_view.h"
+#include "absl/strings/strip.h"  // IWYU pragma: keep
 #include "third_party/zynamics/binexport/util/process.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
 
diff --git a/util/filesystem.h b/util/filesystem.h
index 4fa9459..45fcde1 100644
--- a/util/filesystem.h
+++ b/util/filesystem.h
@@ -20,9 +20,9 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 #ifdef _WIN32
diff --git a/util/filesystem_test.cc b/util/filesystem_test.cc
index f197fb1..15ae4f7 100644
--- a/util/filesystem_test.cc
+++ b/util/filesystem_test.cc
@@ -19,7 +19,7 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/status_matchers.h"
 
 namespace security::binexport {
diff --git a/util/format.cc b/util/format.cc
index 04e180f..e1ded02 100644
--- a/util/format.cc
+++ b/util/format.cc
@@ -4,9 +4,9 @@
 #include <cinttypes>
 #include <cstdio>
 
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/time/time.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/time/time.h"
 
 namespace security::binexport {
 
diff --git a/util/format.h b/util/format.h
index ca7b35a..f4baa77 100644
--- a/util/format.h
+++ b/util/format.h
@@ -17,7 +17,7 @@
 
 #include <string>
 
-#include "third_party/absl/time/time.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 namespace security::binexport {
diff --git a/util/idb_export.cc b/util/idb_export.cc
index 6aa265f..7939251 100644
--- a/util/idb_export.cc
+++ b/util/idb_export.cc
@@ -18,20 +18,20 @@
 #include <memory>
 #include <thread>  // NOLINT
 
-#include "third_party/absl/base/attributes.h"
-#include "third_party/absl/container/flat_hash_map.h"
-#include "third_party/absl/container/flat_hash_set.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/match.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/str_join.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/synchronization/mutex.h"
-#include "third_party/absl/time/time.h"
+#include "absl/base/attributes.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/container/flat_hash_set.h"
+#include "absl/memory/memory.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/str_join.h"
+#include "absl/strings/string_view.h"
+#include "absl/synchronization/mutex.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/util/process.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
diff --git a/util/idb_export.h b/util/idb_export.h
index f7aecee..ed51da2 100644
--- a/util/idb_export.h
+++ b/util/idb_export.h
@@ -18,10 +18,10 @@
 #include <memory>
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/synchronization/mutex.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+#include "absl/synchronization/mutex.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 namespace security::binexport {
diff --git a/util/logging.cc b/util/logging.cc
index f237df4..faa16f1 100644
--- a/util/logging.cc
+++ b/util/logging.cc
@@ -24,18 +24,18 @@
 #include <sstream>
 #include <thread>  // NOLINT
 
-#include "third_party/absl/base/internal/sysinfo.h"
-#include "third_party/absl/log/initialize.h"
-#include "third_party/absl/log/log.h"
-#include "third_party/absl/log/log_sink.h"
-#include "third_party/absl/log/log_sink_registry.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_format.h"
-#include "third_party/absl/strings/string_view.h"
-#include "third_party/absl/synchronization/mutex.h"
-#include "third_party/absl/time/clock.h"
-#include "third_party/absl/time/time.h"
+#include "absl/base/internal/sysinfo.h"
+#include "absl/log/initialize.h"
+#include "absl/log/log.h"
+#include "absl/log/log_sink.h"
+#include "absl/log/log_sink_registry.h"
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/string_view.h"
+#include "absl/synchronization/mutex.h"
+#include "absl/time/clock.h"
+#include "absl/time/time.h"
 #include "third_party/zynamics/binexport/util/process.h"
 
 namespace security::binexport {
diff --git a/util/logging.h b/util/logging.h
index fa387d9..de57dae 100644
--- a/util/logging.h
+++ b/util/logging.h
@@ -20,9 +20,9 @@
 #include <string>
 #include <thread>  // NOLINT
 
-#include "third_party/absl/base/log_severity.h"
-#include "third_party/absl/log/log_sink.h"
-#include "third_party/absl/status/status.h"
+#include "absl/base/log_severity.h"
+#include "absl/log/log_sink.h"
+#include "absl/status/status.h"
 
 namespace security::binexport {
 
diff --git a/util/process.cc b/util/process.cc
index a7467fa..667450e 100644
--- a/util/process.cc
+++ b/util/process.cc
@@ -41,11 +41,11 @@
 #include <cstring>
 #include <locale>  // IWYU pragma: keep
 
-#include "third_party/absl/base/attributes.h"  // IWYU pragma: keep
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/strings/ascii.h"
-#include "third_party/absl/strings/str_cat.h"
-#include "third_party/absl/strings/str_replace.h"  // IWYU pragma: keep
+#include "absl/base/attributes.h"  // IWYU pragma: keep
+#include "absl/status/status.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_replace.h"  // IWYU pragma: keep
 #include "third_party/zynamics/binexport/util/filesystem.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
 
diff --git a/util/process.h b/util/process.h
index aba05ac..bbc1ec6 100644
--- a/util/process.h
+++ b/util/process.h
@@ -18,9 +18,9 @@
 #include <string>
 #include <vector>
 
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/string_view.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 namespace security::binexport {
diff --git a/util/process_test.cc b/util/process_test.cc
index d0aedc7..6dcd563 100644
--- a/util/process_test.cc
+++ b/util/process_test.cc
@@ -19,7 +19,7 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/status_matchers.h"
 
 namespace security::binexport {
diff --git a/util/status_macros.h b/util/status_macros.h
index 64e8afa..36d51ad 100644
--- a/util/status_macros.h
+++ b/util/status_macros.h
@@ -18,9 +18,9 @@
 #ifndef UTIL_STATUS_MACROS_H_
 #define UTIL_STATUS_MACROS_H_
 
-#include "third_party/absl/base/optimization.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
+#include "absl/base/optimization.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
 
 // Internal helper for concatenating macro values.
 #define NA_MACROS_IMPL_CONCAT_INNER_(x, y) x##y
diff --git a/util/status_macros_test.cc b/util/status_macros_test.cc
index 7e66023..885afcc 100644
--- a/util/status_macros_test.cc
+++ b/util/status_macros_test.cc
@@ -18,10 +18,10 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/memory/memory.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/strings/str_cat.h"
+#include "absl/memory/memory.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/str_cat.h"
 #include "third_party/zynamics/binexport/util/status_matchers.h"
 
 namespace not_absl {
diff --git a/util/status_matchers.h b/util/status_matchers.h
index 23ed2dc..3ad6534 100644
--- a/util/status_matchers.h
+++ b/util/status_matchers.h
@@ -19,9 +19,9 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "third_party/absl/status/status.h"
-#include "third_party/absl/status/statusor.h"
-#include "third_party/absl/types/optional.h"
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/types/optional.h"
 #include "third_party/zynamics/binexport/util/status_macros.h"
 
 #define NA_ASSERT_OK_AND_ASSIGN(lhs, rexpr) \
diff --git a/virtual_memory.h b/virtual_memory.h
index caebcf7..326f748 100644
--- a/virtual_memory.h
+++ b/virtual_memory.h
@@ -19,8 +19,8 @@
 #include <map>
 #include <vector>
 
-#include "third_party/absl/container/btree_map.h"
-#include "third_party/absl/container/flat_hash_map.h"
+#include "absl/container/btree_map.h"
+#include "absl/container/flat_hash_map.h"
 #include "third_party/zynamics/binexport/util/types.h"
 
 class AddressSpace {
diff --git a/writer.h b/writer.h
index 4416ecf..97476f1 100644
--- a/writer.h
+++ b/writer.h
@@ -15,7 +15,7 @@
 #ifndef WRITER_H_
 #define WRITER_H_
 
-#include "third_party/absl/status/status.h"
+#include "absl/status/status.h"
 #include "third_party/zynamics/binexport/address_references.h"
 #include "third_party/zynamics/binexport/comment.h"
 #include "third_party/zynamics/binexport/instruction.h"
diff --git a/x86_nop.h b/x86_nop.h
index 43f0dfb..bab13ba 100644
--- a/x86_nop.h
+++ b/x86_nop.h
@@ -17,7 +17,7 @@
 
 #include <cstddef>
 
-#include "third_party/absl/strings/string_view.h"
+#include "absl/strings/string_view.h"
 
 // Returns true if the instruction starting at the first byte in "bytes" is a
 // NOP instruction. NOPs have been taken from the GNU assembler
